scripts/play.ts

```diff
@@ -0,0 +1,380 @@
+    } else {
+      console.info(
+        `Reusing most recent playground: ${chalk.blue(playgroundName)}`,
+      );
+    }
+
+    console.info('  (Use "--new" to create a new playground)');
+    console.info();
+
+    if (args.framework) {
+      console.warn(
+        chalk.yellow`The --framework/-f argument is ignored when a playground already exists.`,
+      );
+    }
+  } else {
+    console.log(`Creating a new playground: ${chalk.blue(playgroundName)}`);
+
+    await createPlayground(playgroundName, playgroundRoot, {
+      framework: args.framework ?? DEFAULT_FRAMEWORK,
+      dryRun: args["dry-run"],
+    });
+  }
+
+  if (args.start) {
+    await exec("yarn", ["dev"], {
+      cwd: playgroundRoot,
+      stdio: "inherit",
+      dryRun: args["dry-run"],
+    });
+  }
+}
+
+/**
+ * Creates a new playground.
+ *
+ * @param name - The name of the playground. It must be unique.
+ * @param root - The directory of the playground. It must be unique.
+ * @param options - Options that determine the function's behavior.
+ */
+async function createPlayground(
+  name: string,
+  root: string,
+  options: {
+    /**
+     * The framework used to bootstrap the playground.
+     */
+    framework: Args["framework"];
+
+    /**
+     * If `true`, commands are not executed.
+     */
+    dryRun?: boolean;
+  },
+) {
+  switch (options.framework ?? "next") {
+    case "next": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/nextjs-starter-prismic-minimal-ts.git",
+        root,
+        { dryRun: options.dryRun },
+      );
+
+      await updateJSONFile<PackageJSON>(
+        path.join(root, "package.json"),
+        { scripts: { "dev:website": "next dev --port=8001" } },
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    case "sveltekit": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/sveltekit-starter-prismic-minimal.git",
+        root,
+        { dryRun: options.dryRun },
+      );
+
+      await updateJSONFile<PackageJSON>(
+        path.join(root, "package.json"),
+        { scripts: { "dev:website": "vite dev --port=8001" } },
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    default: {
+      console.error(chalk.red`Framework not supported: ${options.framework}`);
+      process.exit(1);
+    }
+  }
+
+  if (!options.dryRun) {
+    await fs.rm(path.join(root, ".git"), { recursive: true });
+    await fs.rm(path.join(root, "package-lock.json"));
+  }
+
+  await exec("yarn", ["add", "--dev", "concurrently"], {
+    cwd: root,
+    stdio: "inherit",
+    dryRun: options.dryRun,
+  });
+
+  await updateJSONFile<PackageJSON>(
+    path.join(root, "package.json"),
+    {
+      name,
+      scripts: {
+        dev: 'concurrently --prefix-colors auto "yarn:dev:website" "yarn:dev:slicemachine"',
+        "dev:slicemachine": `SM_ENV=staging ../../packages/start-slicemachine/bin/start-slicemachine.js`,
```

Comment:
Can we use [`cross-env`](https://www.npmjs.com/package/cross-env) here? Could be added alongside `concurrently` above?

Reply:
Yep, good idea!